desktop/core/src/desktop/js/reactComponents/Toolbar/ToolbarDivider/ToolbarDivider.tsx

```diff
@@ -0,0 +1,39 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Divider } from 'antd';
+import React, { FunctionComponent } from 'react';
+
+import './ToolbarDivider.scss';
+
+export interface ToolbarDividerProps {
+  testId?: string;
+}
+
+const defaultProps = {
+  testId: 'hue-toolbar-divider'
+};
+
+const ToolbarDivider: FunctionComponent<ToolbarDividerProps> = ({
+  testId
```

Comment:
I see you're adding const defaultProps in some places... With TS you assign the defaults right away in the args i.e.

... = ({ testId = 'hue-toolbar-divider' })

Is there a benefit of setting it on the component? I.e. ToolbarDivider.defaultProps = defaultProps.

Reply:
There are some edge case benefits like ability to access the defaults outside the component or access a default internally even though the prop has gotten a value. But, now I just learned that defaultProps is about to become deprecated so I'll stop using it and update the code. (I don't think we need TS to use the object destructuring syntax though)